index.markdown

```diff
@@ -1053,7 +1055,7 @@ things like:
 dxw will always pay expenses which are:
 
 * **Necessary**: We only expense things we need in order to be able to complete
-  our work
+  our workte
```

Comment:
this doesn't seem right?

Reply:
Something was happening with the interface being really slow and my search not updating. I think I might switch back to using a text editor locally for myself.